tests/test_utils.py

```diff
@@ -17,10 +17,16 @@ class TestUtils:
     @pytest.mark.parametrize(
         "argm,expectation",
         [
-            ("ocm00012345", "12345"),
-            ("ocn00012346", "12346"),
-            ("on000012347", "12347"),
-            (" ocm00012348", "12348"),
+            ("ocm00054321", "54321"),
+            ("ocm11111", "11111"),
+            ("ocn123456789", "123456789"),
+            ("on1234567890", "1234567890"),
+            ("on12345678901", "12345678901"),
+            (" ocn000111111", "111111"),
+            ("00012345", "12345"),
+            ("11111", "11111"),
+            ("(OCoLC)00012345", "12345"),
```

Comment:
Just a comment: system generated OCLC #s in 035 don't have any leading zeros. Still good your tests include them since we have seen these manually added to the records in the wild.